openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -101,6 +101,8 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
 
     input {
       display: block;
+      padding-top: 0;
```

Comment:
:ship:

I like it, I think fixing the general rule here is more likely to accidentally fix this exact situation in other places than break them.